runtime/lua/vim/treesitter/dev.lua

```diff
@@ -226,7 +226,10 @@ function TSTreeView:draw(bufnr)
         text = string.format('(%s', item.node:type())
       end
     else
-      text = string.format('"%s"', item.node:type():gsub('\n', '\\n'):gsub('"', '\\"'))
+      text = string.format(
+        '"%s"',
+        item.node:type():gsub('\\', '\\\\'):gsub('\n', '\\n'):gsub('"', '\\"')
+      )
```

Comment:
it's probably time to use an actual pattern,

```suggestion
      text = string.format('"%s"', item.node:type()::gsub('["\\\n]', '\\%1'))
```

Reply:
Sadly it doesn't work, when it hits a newline instead of spitting out `\\n` it gives
```
\

```
(a backslash concatenated with a new line)

Reply:
ok then worse case:

    text = string.format('"%s"', item.node:type():gsub('\n', '\\n'):gsub('["\\]', '\\%1'))